samples/bluetooth/peripheral_hr/src/main.c

```diff
@@ -152,6 +207,25 @@ int main(void)
 
 		/* Battery level simulation */
 		bas_notify();
+
+		if (atomic_test_and_clear_bit(state, STATE_CONNECTED)) {
+			/* Connected callback executed */
+
+		} else if (atomic_test_and_clear_bit(state,
+						     STATE_DISCONNECTED)) {
+#if defined(CONFIG_BT_EXT_ADV)
+			printk("Starting Extended Advertising "
+			       "(connectable and non-scannable)\n");
+			err = bt_le_ext_adv_start(adv,
+						  BT_LE_EXT_ADV_START_DEFAULT);
+			if (err) {
+				printk("Failed to start extended advertising "
+				       "set (err %d)\n", err);
+				return 0;
+			}
```

Comment:
Please use the recommended line length of `100`

Reply:
done.